generative/README.md

```diff
@@ -0,0 +1,25 @@
+# MONAI Generative Model Example
+This folder contains examples for training and validating a MONAI Generative Model.
+
+The examples are designed as demonstration to showcase the training process for this type of network using MONAI Generative models. To achieve optimal performance, it is recommended that users to adjust the network and hyper-parameters based on their device and training dataset.
+
+## Installation
+```
+pip install lpips
+pip install git+https://github.com/Project-MONAI/GenerativeModels.git@0.2.0
+```
+
+Or install it from source:
+```
+pip install lpips
+git clone https://github.com/Project-MONAI/GenerativeModels.git
+cd GenerativeModels/
+python setup.py install
+cd ..
+```
+
+## [Brats 3D latent diffusion model](./3d_ldm)
```

Comment:
Folder works great in GitHub, but the link will fail in JupyterLab. It would be better to link the README.md in both folders instead of `./3d_ldm` and `./2d_ldm` here. Thanks!

Reply:
Sure! Thank you!